include/deal.II/distributed/shared_tria.h

```diff
@@ -170,11 +170,21 @@ namespace parallel
     template <int dim, int spacedim = dim>
     class Triangulation : public dealii::parallel::Triangulation<dim,spacedim>
     {
+    public:
+
+      const std::vector<types::subdomain_id> &get_true_subdomain_ids_of_cells() const;
```

Comment:
Please add a 'dummy' description here and for the next function. If done, please merge yourself.